Tasks/codepush-release-cordova/codepush-release-cordova.js

```diff
@@ -83,13 +81,21 @@ function performDeployTask(accessKey, appName, appStoreVersion, platform, deploy
         console.error("Access key required");
         tl.setResult(1, "Access key required");
     }
-  
+
     // Ensure all other users are logged out.
     ensureLoggedOut();
-  
+
     // Log in to the CodePush CLI.
     executeCommandAndHandleResult("login", /*positionArgs*/ null, { accessKey: accessKey });
 
+    // Try to find cordova and prepare the environment if not found. 
+    if (!which("cordova")) {
+        console.log("cordova cli not found. Installing...");
+        exec("npm install cordova");
+        var originalPath = process.env["PATH"];
+        process.env["PATH"] = path.join(process.cwd(), "node_modules", ".bin") + (process.platform == "win32" ? ";" : ":") + originalPath;
```

Comment:
Well I guess you could append instead of prepend the path if you want it to default to the global one, but I think its very low pri.